openmetadata-ui/src/main/resources/ui/src/rest/index.ts

```diff
@@ -12,11 +12,71 @@
  */
 
 import axios from 'axios';
+import {
+  ACTIVE_DOMAIN_STORAGE_KEY,
+  DEFAULT_DOMAIN_VALUE,
+} from 'constants/constants';
+import { isEmpty } from 'lodash';
 import Qs from 'qs';
 
+let requestInterceptor: number | null = null;
+
 const axiosClient = axios.create({
   baseURL: '/api/v1',
   paramsSerializer: (params) => Qs.stringify(params, { arrayFormat: 'comma' }),
 });
 
+export const initializeAxiosInterceptors = async () => {
```

Comment:
We already have a request interceptor in the auth provider, so shouldn't we utilize that?

Reply:
Good spot ! 
Forgot about the auth provider interceptor. moved there